shared-libs/transitions/test/unit/lib/muting_utils.js

```diff
@@ -7,6 +7,7 @@ const utils = require('../../../src/lib/utils');
 const moment = require('moment');
 const infodoc = require('@medic/infodoc');
 
+chai.should();
```

Comment:
Several of the [existing tests below](https://github.com/medic/cht-core/blob/2ba6220ba3f748c708cee0ed6e1105114839122f/shared-libs/transitions/test/unit/lib/muting_utils.js#L1692) were using `should` without ever initializing it in this file. It worked when all the tests were run together, but would fail if this file was run by itself.

Reply:
Oh, I see. Thanks for explaining. The pains of mixed assertions :(